apps/frontend/src/pages/analytics/libs/components/analytics-table/analytics-table.tsx

```diff
@@ -0,0 +1,37 @@
+import { Table } from "~/libs/components/components.js";
+import { type ActivityLogGetAllItemResponseDto } from "~/modules/activity-logs/activity-logs.js";
+
+import {
+	getAnalyticsColumns,
+	getAnalyticsRows,
+	getDateRange,
+} from "../../helpers/helpers.js";
+import { type AnalyticsRow } from "../../types/types.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	activityLogs: ActivityLogGetAllItemResponseDto[];
+	dateRange: [Date, Date];
+};
+
```

Comment:
I might be wrong, but I think it might be convenient to put the table on the page directly, as we won't have any other tables there.